src/imitation/util/video_wrapper.py

```diff
@@ -75,3 +83,70 @@ def close(self) -> None:
             self.video_recorder.close()
             self.video_recorder = None
         super().close()
+
+
+def record_and_save_video(
+    output_dir: str,
+    policy: policies.BasePolicy,
+    eval_venv: vec_env.VecEnv,
+    video_kwargs: Optional[Mapping[str, Any]] = None,
+    logger: Optional[sb_logger.Logger] = None,
+) -> None:
+    video_dir = osp.join(output_dir, "videos")
+    video_venv = VideoWrapper(
+        eval_venv,
+        directory=video_dir,
+        **(video_kwargs or dict()),
+    )
+    sample_until = rollout.make_sample_until(min_timesteps=None, min_episodes=1)
+    # video.{:06}.mp4".format(VideoWrapper.episode_id) will be saved within
+    # rollout.generate_trajectories()
+    rollout.generate_trajectories(policy, video_venv, sample_until)
```

Comment:
For some reason I was expecting that the video that would be saved would be one of the real training trajectories instead of a newly sampled one.